cpp/src/parquet/column_reader_benchmark.cc

```diff
@@ -0,0 +1,99 @@
+#include "benchmark/benchmark.h"
+#include "parquet/column_page.h"
+#include "parquet/column_reader.h"
+#include "parquet/schema.h"
+#include "parquet/test_util.h"
+#include "parquet/types.h"
+
+namespace parquet {
+
+using parquet::Repetition;
+using parquet::test::MakePages;
+using schema::NodePtr;
+
+namespace benchmark {
+
+static void BM_Skip(::benchmark::State& state) {
+  internal::LevelInfo level_info;
```

Comment:
Please add a brief summary below on instantiation what ranges of values this is checking.

Reply:
I tried to add clarifying comments. Please take a look.